Source/ACE.Server/Network/Managers/SocketManager.cs

```diff
@@ -35,8 +36,8 @@ public static void Initialize()
             listeners[1] = new ConnectionListener(host, ConfigManager.Config.Server.Network.Port + 1);
             log.Info($"Binding ConnectionListener to {host}:{ConfigManager.Config.Server.Network.Port + 1}");
 
-            listeners[0].Start();
-            listeners[1].Start();
+            new Thread(new ThreadStart(() => { listeners[0].Start(); })) { Name = $"{listeners[0].Socket.LocalEndPoint} Listener" }.Start();
```

Comment:
The listeners do not need to be created on new threads. The current thread they're on doesn't even blip on the CPU usage levels.

Please revert SocketManager changes.